main/input_default.cpp

```diff
@@ -1176,3 +1176,121 @@ int InputDefault::get_joy_axis_index_from_string(String p_axis) {
 	}
 	ERR_FAIL_V(-1);
 }
+
+Array InputDefault::get_connected_trackers(TrackerType p_type) {
+	Array ret;
+	Map<int, Tracker>::Element *elem = trackers.front();
+	while (elem) {
+		if (((elem->get().type == p_type) || (p_type == TRACKER_ANY)) && (elem->get().type != TRACKER_UNKNOWN)) {
+			ret.push_back(elem->key());
+		}
+		elem = elem->next();
+	}
+	return ret;
+};
+
+Input::TrackerType InputDefault::get_tracker_type(int p_idx) const {
+	_THREAD_SAFE_METHOD_
+	return trackers[p_idx].type;
+};
+
+String InputDefault::get_tracker_name(int p_idx) const {
+	_THREAD_SAFE_METHOD_
+	return trackers[p_idx].name;
+};
+
+Transform InputDefault::get_tracker_transform(int p_idx) const {
+	_THREAD_SAFE_METHOD_
+	return trackers[p_idx].transform;
+};
+
+int InputDefault::add_tracker(TrackerType p_type, String p_name) {
+	int new_tracker_id = 1;
+	Tracker new_tracker;
+
+	new_tracker.type = p_type;
+	new_tracker.name = p_name;
+
+	while (trackers.find(new_tracker_id) != NULL) {
+		new_tracker_id++;
+	};
+
+	trackers.insert(new_tracker_id, new_tracker);
+
+	///TODO: send out event of adding a tracker
```

Comment:
Ah thanks for the reminder, I totally forgot about that. Yes that is exactly what I need to add, I'll do that tonight :)